bin/image_insert.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+import sqlalchemy as sa
+from tqdm import tqdm
+import sys
+import argparse
+
+def insert_image(person, image_file, image_type):
+    engine = sa.create_engine('sqlite:///test.db', echo=False)
+    metadata = sa.MetaData()
+
+    person_tbl = sa.Table('person', metadata, autoload=True, autoload_with=engine)
+    img_tbl    = sa.Table('img'   , metadata, autoload=True, autoload_with=engine)
+    points_tbl = sa.Table('points', metadata, autoload=True, autoload_with=engine)
+
+
+    person_sel = person_tbl.select(person_tbl.c.identifier == person)
+    person = engine.execute(person_sel).fetchone()
+
+    if not person:
+        sys.stderr.write("Can't find anyone in the database with the id {}\n".format(person))
```

Comment:
Ah, right, I kind of depend on the insert qual data to make that work. Sorry my bad. A quick fix for now is to run the insert script on that branch (that now should work, fingers crossed) and then try here with, for example, sci001.

In the meantime I will add another method for adding persons to the database (that will probably not be in this branch though, unfortunately).